astropy/tests/runner.py

```diff
@@ -462,6 +462,12 @@ def parallel(self, parallel, kwargs):
             number of CPUs.  If parallel is negative, it will use the all
             the cores on the machine.  Requires the ``pytest-xdist`` plugin.
         """
+        try:
+            from xdist import plugin
+        except ImportError:
+            raise SystemError(
+                "running tests in parallel requires the pytest-xdist package")
+
```

Comment:
Ah but this should be inside the if statement below, otherwise it will fail even if not specifying the parallel option